### PR TITLE
Task Estimation in Scrum

### PR DESCRIPTION
**Pull request by VictorIbikunle**
…ill include more articles/blogs to fnd similarities in things to do with task estimation in scrum. But this is a start

### CODE REVIEW

File: TaskEstimation.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Task Estimation in Scrum
+
+Key Concepts:
+1.Sprint planning:
```
Comment (danieljagun) [inline TaskEstimation.md]: Add a capital P

Comment (VictorIbikunle) [reply TaskEstimation.md:4]: noted

File: TaskEstimation.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+This is a crucial part in a scrum meeting as this is where the team collaberates and determins whcich tasks can be done in the current sprint. The estimation is based off the resources and time that they have for the sprint. This practice would highlight communication and teamwork skills. According to wrike.com( https://www.wrike.com/scrum-guide/scrum-sprint-planning/) it states that the main importance of  sprint planning is "Sets clear defined goals, creates a communication platform and collaberation".
```
Comment (danieljagun) [inline TaskEstimation.md]: Spelling error - "determins", "whcich", "collaberates"

remove extra space and change to "Sprint Planning"

reword the last line - it states that the main importance of  sprint planning is "Sets clear defined goals, creates a communication platform and collaberation".

Remove links and maybe add to end of your part?

File: TaskEstimation.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+2.User Stories:
+User Stories provides an informal, natural language description of a feature of the software or product from the end-user perspective. According to this article user stories are very important because it "Improves transparency and Highest value is delivered"(https://www.simplilearn.com/tutorials/agile-scrum-tutorial/user-stories) without user stories there could be confusion amongst the team as there wont be anything with a clear and cut description of the tasks.
+
+
+Challenges and Skills:
+1.Effort of peers: If certain members within the team are not "pulling their weight" it could affect the group as a whole and can lead to several disagreements and could affect the team as a whole. In this article it briefly describes a scenario but the key thing is from here that the slack of one person was creating frustration "Frustrated, he looked to Ann's desk and she was not there. Again"(https://medium.com/serious-scrum/scrum-team-interactions-under-pressure-ac242e7ec110).
```
Comment (CorneliusAkanbi1) [inline TaskEstimation.md]: could elaborate more as to why ann was frustrated in regards to effort of peers.

File: TaskEstimation.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+2.Skill difference:
+Team members have different skills and experience in a group. This can lead to expectations being set and not being met, i believe this can affect the team in a negative way as some people might profuce high quality efforts and others efforts would be mediocre and could affect the overall output of the project.
```
Comment (CorneliusAkanbi1) [inline TaskEstimation.md]: "I " not capitalised

Comment (danieljagun) [reply TaskEstimation.md]: In section 1 capital P
In section 2 capital D
spelling error - profuce

File: TaskEstimation.md (hunk 2 above)
Comment (VictorIbikunle) [reply TaskEstimation.md]: i feel like referencing to the article and leaving the link beside it is a good structure

File: TaskEstimation.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+Task Estimation in Scrum
+
+Key Concepts:
+1.Sprint Planning:
+This is a crucial part in a scrum meeting as this is where the team collaborates  and determines  whcich tasks can be done in the current sprint. The estimation is based off the resources and time that they have for the sprint. This practice would highlight communication and teamwork skills. According to an articel it states that the main importance of sprint planning is "Sets clear defined goals, creates a communication platform and collaberation".
```
Comment (danieljagun) [inline TaskEstimation.md]: remove the two spaces after collaborates & determines

Comment (danieljagun) [inline TaskEstimation.md]: Spelling error - whcich

Comment (danieljagun) [inline TaskEstimation.md]: Spelling error - articel

Comment (danieljagun) [inline TaskEstimation.md]: Spelling error - collaberation

File: TaskEstimation.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+2.User Stories:
+User Stories provides an informal, natural language description of a feature of the software or product from the end-user perspective. According to this article user stories are very important because it "Improves transparency and Highest value is delivered". Without user stories there could be confusion amongst the team as there wont be anything with a clear and cut description of the tasks.
```
Comment (danieljagun) [inline TaskEstimation.md]: spelling error - provides

Comment (danieljagun) [inline TaskEstimation.md]: spelling error - wont

File: TaskEstimation.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+
+Challenges and Skills:
+1.Effort of peers: If certain members within the team are not "pulling their weight" it could affect the group as a whole and can lead to several disagreements and could affect the team as a whole. In this article it briefly describes a scenario but the key thing is from here that the slack of one person was creating frustration "Frustrated, he looked to Ann's desk and she was not there again". Not putting in effort can bring down the team's morale, increase workload and prevent the team from hitting their target.
```
Comment (danieljagun) [inline TaskEstimation.md:11]: just say can lead to disagreements

File: TaskEstimation.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+2.User Stories:
+User Stories provides an informal, natural language description of a feature of the software or product from the end-user perspective. According to this article user stories are very important because it "Improves transparency and Highest value is delivered". Without user stories there could be confusion amongst the team as there wont be anything with a clear and cut description of the tasks.
+
+
+Challenges and Skills:
+1.Effort of peers: If certain members within the team are not "pulling their weight" it could affect the group as a whole and can lead to several disagreements and could affect the team as a whole. In this article it briefly describes a scenario but the key thing is from here that the slack of one person was creating frustration "Frustrated, he looked to Ann's desk and she was not there again". Not putting in effort can bring down the team's morale, increase workload and prevent the team from hitting their target. 
+
+2.Skill difference:
```
Comment (danieljagun) [inline TaskEstimation.md:13]: capital D

File: TaskEstimation.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+Team members have different skills and experience in a group. This can lead to expectations being set and not being met, I believe this can affect the team in a negative way as some people might produce high quality efforts and others efforts would be mediocre and could affect the overall output of the project.
```
Comment (danieljagun) [inline TaskEstimation.md:14]: other efforts may be mediocre*

**Review by danieljagun — APPROVED** [review]

**Review by CorneliusAkanbi1 — APPROVED** [review]

**Pull request merged**